rest-api/src/main/resources/definitions/study.yml

```diff
@@ -36,6 +36,13 @@ properties:
     consentNotificationEmailVerified:
         description: True if the consent notification email is verified. False if not.
         type: boolean
+    installLinks:
+        type: object
+        description: |
+            A map between operating system names, and a link to send via SMS to acquire the study's app. This can be
+            either to an app store, or an intermediate web page that will route to a final app or appstore.
```

Comment:
"This can be either to an app store (URLs mapped to the keys "iOS" or "Android"), or an intermediate web page that will route to a final app or appstore (URLs mapped to the key "Universal"). iOS and Android take precedence over the Universal URL."

Reply:
Updated description.